main.go

```diff
@@ -23,6 +29,8 @@ var (
 	helpTab = strings.Repeat(" ", helpTabWidth)
 	// version is overwritten by ci/build.sh.
 	version string
+	res     string
+	dir     string
```

Comment:
dir global doesn't seem to be used anywhere, I don't understand the purpose of res and I can't find where it's used after being set either.

Reply:
Will take a look at this, if not needed to be declared globally, i will move it to the necessary function where it is needed,